packages/errors/src/fuel-error.ts

```diff
@@ -37,7 +37,7 @@ export class FuelError extends Error {
     code: ErrorCode,
     message: string,
     metadata: Record<string, unknown> = {},
-    rawError: unknown = {}
+    rawError: unknown = null
```

Comment:
@Dhaiwat10 Why we had to change this to null?

Reply:
Here is the reasoning: https://github.com/FuelLabs/fuels-ts/pull/2902#issuecomment-2288013444

Reply:
I think I'd prefer `undefined`.

```ts
 rawError?: unknown
```

Reply:
I usually tend towards a `null` value when representing the absence of an object value.

I don't have a strong opinion on this.